modules/installation-ibm-cloud-tested-machine-types.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-private.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc
+// installing/installing_ibm_cloud_public/installing-ibm-cloud-vpc.adoc
+
+[id="installation-ibm-cloud-tested-machine-types_{context}"]
```

Comment:
Checked that section `Tested instance types for IBM Cloud` is added starting from 4.15 doc, https://docs.openshift.com/container-platform/4.15/installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.html#installation-ibm-cloud-tested-machine-types_installing-ibm-cloud-customizations

Do we need this adoc on 4.14 release? cc: @MayXuQQ

Reply:
I prefer to add this section to 4.14 release